index.bs

```diff
@@ -10780,10 +10780,18 @@ This object's relevant [=Realm=] must be the same as that of the [=named constru
             and with argument count |n|.
         1.  Let &lt;|constructor|, |values|&gt; be the result of passing |S| and
             |args| to the [=overload resolution algorithm=].
-        1.  Let |R| be the result of performing the actions listed in the description of
-            |constructor| with |values| as the argument values.
-        1.  Return the result of [=converted to an ECMAScript value|converting=]
-            |R| to an ECMAScript [=interface type=] value |I|.
+        1.  Let |object| be the result of [=internally create a new object implementing the
+            interface|internally creating a new object implementing=] |I|, with |realm| and
+            {{NewTarget}}.
+        1.  Perform the actions listed in the description of |constructor|
+            with |values| as the argument values
+            and |object| as the <emu-val>this</emu-val> value.
+            Rethrow any exceptions.
+        1.  Let |O| be the result of [=converted to an ECMAScript value|converting=] |object|
+            to an ECMAScript [=interface type=] value |I|.
+        1.  Assert: |O| is an object that implements |I|.
```

Comment:
link implements?

Reply:
There's no formal definition yet; I'm planning to add one soon, though.